src/test/java/games/strategy/ui/SwingActionTest.java

```diff
@@ -48,4 +51,16 @@ public void testInvokeAndWait(@Mock final Runnable action) throws Exception {
       verify(action, times(2)).run();
     });
   }
+
+  @Test
+  public void testInvokeNowOrLater() {
+    final CountDownLatch latch = new CountDownLatch(1);
+    final Runnable action = () -> {
+      latch.countDown();
+    };
+
+    SwingAction.invokeNowOrLater(action);
+
+    assertTimeoutPreemptively(Duration.ofSeconds(5L), () -> latch.await());
```

Comment:
This lambda can be shortened as well.

Reply:
Hmm, the compiler thinks `latch::await` is ambiguous in this context.  It can't decide between the `assertTimeoutPreemptively()` overload that accepts an `Executable` and the one that accepts a `ThrowingSupplier`.  Not sure I understand what's triggering that. 😕 

Regardless, the current lambda ends up being shorter than casting the method reference: `(Executable) latch::await`, so I'm going to leave it as-is for now.